src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/web/IndexPageTests.kt

```diff
@@ -1,58 +1,15 @@
 package org.vaccineimpact.orderlyweb.tests.integration_tests.tests.web
 
-import org.assertj.core.api.Assertions
-import org.jooq.impl.DSL
-import org.jsoup.Jsoup
 import org.junit.Test
-import org.vaccineimpact.orderlyweb.db.JooqContext
-import org.vaccineimpact.orderlyweb.db.Tables.*
 import org.vaccineimpact.orderlyweb.models.Scope
 import org.vaccineimpact.orderlyweb.models.permissions.ReifiedPermission
 import org.vaccineimpact.orderlyweb.tests.integration_tests.tests.IntegrationTest
 
 class IndexPageTests : IntegrationTest()
 {
-    private val readReports = setOf(ReifiedPermission("reports.read", Scope.Global()))
-    private val reviewReports =  setOf(ReifiedPermission("reports.review", Scope.Global()))
-    private val indexUrl = "/"
-
-    @Test
-    fun `report reviewers can see reports whose versions are all unpublished`()
```

Comment:
The logic in these deleted tests is covered by unit tests.

Reply:
That's true - I guess putting in an integration test checks that all the security stuff is working together, though you don't want to have to repeat that check for every permission! So we should maybe keep a couple of security integration tests for safety, and also make sure that all the classes are properly unit tested. I don't think there are currently any unit tests for DirectActionContext.hasPermission for example.

Reply:
Yes actually I think DirectActionContext is completely untested - see the branch that introduces code coverage reports. Should spawn a few tickets for getting coverage up to 100%